src/sql/workbench/services/tableDesigner/browser/tableDesignerComponentInput.ts

```diff
@@ -580,4 +607,13 @@ export class TableDesignerComponentInput implements DesignerComponentInput {
 			this._viewModel[property] = {};
 		}
 	}
+
+	private createEditActionEvent(edit: DesignerEdit): ITelemetryEvent {
+		const editAction = this._adsTelemetryService.createActionEvent(TelemetryView.TableDesigner, edit.type).withAdditionalProperties({
+			objectType: edit.path[0],
```

Comment:
1. I think we only planned to capture the add and removal operations, right? 
2. edit.path[0] will only capture the top level property, If I am trying to add a column to an index, this won't work

Reply:
1. Yes, but doing it generically gives us all actions for free.
2. Do you have a list of scenarios of things you can do right now in Table Designer? I'd like to know all of them so I can test all.